tests/e2e/test_chaperone.js

```diff
@@ -74,6 +74,7 @@ const envoyOpts = {
   hosted_port_number: 0,
   hosted_app: {
     servicelogger_id: HOSTED_APP_SERVICELOGGER_INSTALLED_APP_ID,
+    // TODO: Why do we need this?
```

Comment:
Is this still a question?

Reply:
Yes, Could you answer it?

Reply:
This allows us to quickly edit the app that we would like to test.  For example, we eventually want to test against the mock dna, which could be one of the apps provided here.  I've also used it in the past to test out calling different dnas or a different version of a dna quickly.

Reply:
Well, I get this for unit tests but for e2e tests, we do not use a mock conductor. And also we need to pre-install a read-only instance for the installation of a happ during sign-up work.

Reply:
So in the config, the path expects the path to where the app would already be physically installed, which means that it would simply just need to be registered (which was what the `installDnas` function was doing here: https://github.com/Holo-Host/holo-envoy/pull/51/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L308

Reply:
Ah, but yes, I see what you're saying - it is true if we needed a read-only instance, we would need to call the admin install function after registering it as well...